scripts/install.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
```

Comment:
Also I saw this is the best use neither `env...` nor `./bash.` is good => this one is the best usecase